src/index.html

```diff
@@ -5,8 +5,371 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#logo">
+        <img
+          src="./images/logo.png"
+          class="header__img"
+          alt="Moyo logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="card-img"
+          class="card__img"
+        >
+
+        <div class="card__info">
+          <div class="card__overview">
+            <div class="card__titles">
+              <h1 class="card__title">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h1>
```

Comment:
This tag is usually only used once on the page in one place, so it would be better if you use another level heading everywhere
```suggestion
              <h1 class="card__title">
                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
              </h1>
```